### PR TITLE
Operators

### PR DESCRIPTION
**Pull request by OleksandrZheleznyi**

### CODE REVIEW

File: Operators/1/Block.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        public static bool operator !=(Block b1, Block b2)
+        {
+            return !b1.Equals(b2);
+        }
+
+        public override bool Equals(object item)
+        {
+            if (item is Block)
+            {
+                if (((Block)item)._a == _a &&
```
Comment (OleksandrIvanovTheBest) [inline Operators/1/Block.cs]: you are converting item to Book 4 times. Do it ones. Think what operator is better to use.

File: Operators/3/Date.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            this.month = month;
+            this.day = day;
+        }
+
+        public static Date operator -(Date d1, Date d2)
+        {
+            return new Date(d1.year - d2.year, d1.month - d2.month, d1.day - d2.day);
+        }
+
+        public static Date operator +(Date d1, int days)
```
Comment (OleksandrIvanovTheBest) [inline Operators/3/Date.cs:28]: What this method does ?

File: Operators/2/House.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        {
+            House other = (House)this.MemberwiseClone();
+            other.width = width;
+            other.height = height;
+            return other;
+        }
+
+        public object Clone()
+        {
+            return new House(width, height) as object;
```
Comment (OleksandrIvanovTheBest) [inline Operators/2/House.cs]: use BinaryStream here

**Review by OleksandrIvanovTheBest — CHANGES_REQUESTED** [review]

File: Operators/3/Date.cs (hunk 2 above)
Comment (OleksandrZheleznyi) [reply Operators/3/Date.cs:28]: It was in the assignment "Using the operator overload mechanism, define the operation of increasing the date by a certain number of days." And I used overload operator "+" .

**Review by OleksandrIvanovTheBest — APPROVED** [review]

**Pull request merged**